test/micro/org/openjdk/bench/java/lang/StringBuilders.java

```diff
@@ -241,17 +241,68 @@ public String toStringCharWithBool8() {
 
 
     @Benchmark
-    public String toStringCharWithFloat8() {
-        StringBuilder result = new StringBuilder();
-        result.append(113.110F);
-        result.append(156456.36435637F);
-        result.append(65436434.64632F);
-        result.append(42654634.64540F);
-        result.append(63464351.64537F);
-        result.append(634564.645711F);
-        result.append(64547.64311F);
-        result.append(4763456341.64531F);
-        return result.toString();
+    public int appendWithFloat8Latin1() {
```

Comment:
It would be interesting with a mixed micro to explore effects of increased number of implementors (looping over an array with `sbLatin1` and `sbUtf16`, and appending both floats and doubles in the same loop) 

`buf.delete(0, buf.length())` could be replaced with `buf.setLength(0)` in these. Might reduce a bit of overhead and focus micros .